test/semantics/common.sh

```diff
@@ -35,5 +36,7 @@ case $1 in
   (/*) src=$1 ;;
   (*) src=$(dirname $0)/$1 ;;
 esac
+USER_OPTIONS=`sed -n 's/^ *! *OPTIONS: *//p' $src`
+echo $USER_OPTIONS
```

Comment:
Earlier if there were multiple `! OPTIONS:` lines the actual options would be dumped out on multiple lines.
Now they will be listed on the same line separated by a space.  Not saying that it's necessarily wrong, just noting the difference.

Reply:
Okay, thanks! I think it won't influence the results. Earlier it was only writing the options into a file and `cat` it during the actual compilation line, which also ended up with the same line.